help-beta/src/styles/main.css

```diff
@@ -3,3 +3,34 @@
     pointer-events: none;
     cursor: default;
 }
+
+img {
```

Comment:
Can we scope this CSS more precisely to just images in the rendered markdown body of the page? Top-level CSS like this is dangerous since it applies to anything on the page. 

(Also, does `astro` might have CSS classes on their Markdown image component that one can apply this to? That'd be even better, and I kinda expect it does)

Reply:
Scoped the css, but the <Image> component in Astro does not have a fixed class that astro uses. It uses tailwind, which I think leads to a classname with some kind of unique number.